src/main/java/Circle.java

```diff
@@ -0,0 +1,22 @@
+public class Circle extends GeometricObject {
+
+    private String shape = "круг";
```

Comment:
```suggestion
    private static final String CIRCLE_SHAPE = "круг";
    private int radius;
```
same for all other classes that extend GeometricObject

Reply:
done